files/default/neutron-ha-tool.py

```diff
@@ -1058,6 +1025,95 @@ def pick(self):
         )
 
 
+class RemoteRouterNsCleanup(object):
+
+    def __init__(self, host, routerid):
+        self.target_host = host
+        self.namespace = "qrouter-" + routerid
+        self.ssh_client = paramiko.SSHClient()
+        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.ssh_client.load_system_host_keys()
+        self.ssh_client.connect(self.target_host, timeout=10)
+        self.timeout = 10
+        self.netns_del = "ip netns delete " + self.namespace
+        self.netns_pids = "ip netns pids " + self.namespace
+        self.netns_list = "ip netns list"
+
+    def _simple_ssh_command(self, command):
+        # Note, that when get_pty is True, paramiko will never return anything
+        # on the stderr channel, that's why we ignore it here. (stderr output
+        # will endup in the stdout channel)
+        _, stdout, _ = self.ssh_client.exec_command(command,
+                                                    timeout=10,
+                                                    get_pty=True)
+        out_lines = stdout.readlines()
+        rc = stdout.channel.recv_exit_status()
+        return [rc, out_lines]
```

Comment:
Jeje is like Go.

I would use a different approach here: raise an exception if rc is an error code